includes/init_includes/init_ip_blocker.php

```diff
@@ -65,6 +65,9 @@ function ip_blocker_pass($ip)
     return false;
 }
 
+// exit if file is being loaded from Admin
+if (IS_ADMIN_FLAG === true) return;
```

Comment:
I know that I'm being a PITA, but could you wrap that `return` statement with braces?  Other than definition-checking, I'm not a fan of single-line processing statements (there _always_ turns out to be something else that needs to be done).

Reply:
Done